doc/account.md

```diff
@@ -61,18 +61,14 @@ If the account validates, we create the account file from the registration file,
 Passwords
 ---------
 
-Passwords are handled on the clients side. We should never see an unhashed password, unless of course the client sends one in to use, but we won't store that anywhere except their encrpyted account file.
+Passwords are handled on the clients side and then stored their encrypted account file upon completing the sign up procss.
 
 Passwords should be hashed along with the `passwordSalt` we provided them during account signup.
 
-```
     hash(password + passwordSalt)
-```
 
 When sending the password in for authentication, the password should be hashed again and then hashed with the challenge the applictation sent them.
 
-```
     hash(hash(hashedPassword) + challenge)
-```
 
-The hashing of the password is primarily up to the client, as we do nothing with the password except save it in their account file. But the hashing of the challenge and then subsequent hashing after that will be set in `config.json` and will utilize `pbkdf2` on both sides to maintain a consistent login paradigm.
\ No newline at end of file
+The hashing of the password is primarily up to the client, as we only store the password. The hashing of the challenge and subsequent hashing after will be set up in `config.json` and will utilize `pbkdf2` on both sides to maintain a consistent login paradigm.
```

Comment:
What does "primarily up to the client" mean?  Does it mean they can hash however they want?  If so, that is probably incorrect.  Does it mean that we don't dictate?  Then that's wrong.  Perhaps it is that we don't do the hashing?  If so, then it is hashed by the client, but not "primarily" up to them.

Our goal is to tell the client how to hash the password (documentation), then to verify that it is hashed (proper length, characters), and dictate with an iron fist how the verification will be done (documentation + configuration).  We even have to dictate an example and reject things that don't match, such as when the wrong capitalization is used.  5d2e != 5D2E when you are worrying about rehashing stuff, which is what we will do with the challenge.